index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Second Milestone</title>
+    <link rel="stylesheet" href="./stylesheet/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&family=Ubuntu:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
```

Comment:
- [ ] Kindly add back the header you did in your previous task it is still part of your portfolio design you don't need to remove it.
- [ ] Nice working on the `My Works Section`, Kindly put the header (hx) tag and all the projects in a `section` tag for better use of HTML semantics.
- [ ] Also kindly remove the borders from the `see project button` so it matches what you have in the Figma template.